### PR TITLE
Adds CSS frameworks docs in the new structure

### PR DESCRIPTION
**Pull request by craigmc08**
### Description

CSS frameworks page. Some minor changes from the current docs page, but overall similar.

### Select what type of change this PR introduces:

1. [X] **Just code/docs improvement** (no functional change). 
2. [ ] **Bug fix** (non-breaking change which fixes an issue).
3. [ ] **New feature** (non-breaking change which adds functionality).
4. [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

### Update Waspc ChangeLog and version if needed
If you did a bug fix, new feature, or breaking change, that affects waspc, make sure you satisfy the following:
1. [ ] I updated [ChangeLog.md](https://github.com/wasp-lang/wasp/blob/main/waspc/ChangeLog.md) with description of the change this PR introduces.
2. [ ] I bumped waspc version in [waspc.cabal](https://github.com/wasp-lang/wasp/blob/main/waspc/waspc.cabal) to reflect changes I introduced, with regards to the version of the latest wasp release, if the bump was needed.

### CODE REVIEW

File: web/docs/project/css-frameworks.md [hunk 1]
```diff
@@ -1,3 +1,109 @@
 ---
 title: CSS Frameworks
 ---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Tailwind
+
+To enable support for Tailwind in your project, you need to add two config files—[`tailwind.config.cjs`](https://tailwindcss.com/docs/configuration#configuration-options) and `postcss.config.cjs`—to the root directory. With these files present, Wasp installs the necessary dependencies and copies your configuration to the generated project. You can then use [Tailwind CSS directives](https://tailwindcss.com/docs/functions-and-directives#directives) in your CSS and Tailwind classes on your React components.
```
Comment (infomiho) [inline web/docs/project/css-frameworks.md:9]:
```suggestion
To enable support for Tailwind in your project, you need to add two config files — [`tailwind.config.cjs`](https://tailwindcss.com/docs/configuration#configuration-options) and `postcss.config.cjs`— to the root directory. 

With these files present, Wasp installs the necessary dependencies and copies your configuration to the generated project. You can then use [Tailwind CSS directives](https://tailwindcss.com/docs/functions-and-directives#directives) in your CSS and Tailwind classes on your React components.
```

File: web/docs/project/css-frameworks.md [hunk 2]
```diff
@@ -1,3 +1,109 @@
...
+```
+
+:::tip Tailwind not working?
+If you can not use Tailwind after adding the required config files, make sure to restart `wasp start`. This is sometimes needed to ensure that Wasp picks up the changes and enables Tailwind integration.
+:::
+
+### Enabling Tailwind Step-by-Step
+
+:::caution
+Make sure to use the `.cjs` extension for these config files! If you name them with a `.js` extension, Wasp will not enable Tailwind integration.
```
Comment (infomiho) [inline web/docs/project/css-frameworks.md:35]:
```suggestion
Make sure to use the `.cjs` extension for these config files. If you name them with a `.js` extension, Wasp won't detect them.
```

File: web/docs/project/css-frameworks.md [hunk 3]
```diff
@@ -1,3 +1,109 @@
...
+    plugins: {
+      tailwindcss: {},
+      autoprefixer: {},
+    },
+  }
+  ```
+
+3. Import Tailwind into your CSS file. For example, in a new project you might import Tailwind into `Main.css`.
+
+  ```css title="./src/client/Main.css {1-3}"
```
Comment (infomiho) [inline web/docs/project/css-frameworks.md:64]:
```suggestion
  ```css title="./src/client/Main.css" {1-3}
```

**Comment by infomiho** [comment]
Left a couple of minor comments

**Comment by craigmc08** [comment]
@infomiho Addressed comments

**Review by infomiho — APPROVED** [review]
⛴ it

**Pull request merged**